mappings/net/minecraft/entity/ai/brain/task/BreedTask.mapping

```diff
@@ -0,0 +1,11 @@
+CLASS net/minecraft/class_4806 net/minecraft/entity/ai/brain/task/BreedTask
+	FIELD field_22283 targetType Lnet/minecraft/class_1299;
+	FIELD field_22284 breedTime J
+	METHOD <init> (Lnet/minecraft/class_1299;)V
+		ARG 1 targetType
+	METHOD method_24542 getBreedTarget (Lnet/minecraft/class_1429;)Lnet/minecraft/class_1429;
+		ARG 1 animal
+	METHOD method_24546 hasBreedTarget (Lnet/minecraft/class_1429;)Z
+		ARG 1 animal
+	METHOD method_24548 getBreedingTarget (Lnet/minecraft/class_1429;)Ljava/util/Optional;
```

Comment:
What's the difference between `getBreedTarget` and `getBreedingTarget`?

Reply:
Should be `findBreedTarget` indeed!